coingecko/src/endpoint/price.ts

```diff
@@ -0,0 +1,82 @@
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { ExecuteWithConfig, Config } from '@chainlink/types'
+
+export const NAME = 'price'
+
+const customError = (data: any) => {
+  if (Object.keys(data).length === 0) return true
+  return false
+}
+
+const customParams = {
+  base: ['base', 'from', 'coin'],
+  quote: ['quote', 'to', 'market'],
+  coinid: false,
+}
+
+const presetTickers: { [ticket: string]: string } = {
+  COMP: 'compound-governance-token',
+  FNX: 'finnexus',
+  UNI: 'uniswap',
+}
+
+const convertFromTicker = async (config: Config, ticker: string, coinId: string) => {
+  if (typeof coinId !== 'undefined') return coinId.toLowerCase()
+
+  // Correct common tickers that are misidentified
+  if (ticker in presetTickers) {
+    return presetTickers[ticker]
+  }
+
+  const url = 'https://api.coingecko.com/api/v3/coins/list'
```

Comment:
Could we append the `/coins/list` on the `config.baseUrl` instead??

Reply:
Yes!